Validation/RecoTrack/python/TrackValidation_cff.py

```diff
@@ -674,3 +674,21 @@ def _getMVASelectors(postfix):
     tracksValidationTruth +
     trackValidatorLite
 )
+
+## customization for timing
+from Configuration.Eras.Modifier_phase2_timing_cff import phase2_timing
+phase2_timing.toModify( generalTracksFromPV, 
```

Comment:
This changes the default vertex and track  validation to use the 4D by default, while this is not the standard vertex [yet?] for this setup.
Can the 4D validation be added instead of replace the default?

@kpedro88 please check to see if this matches the strategy of phase-2 relvals.
With this change some care will need to be taken to decouple validation of PV used for standard reco (in non-timing workflows) while the production will probably again be running with the phase2_timing enabled.

Reply:
@slava77 that's a good point. Most of the relval workflows don't use `phase2_timing`, so it would only affect the 20034.2 relvals. But maybe this change should be done with `phase2_timing_layer`, where the 4D vertices are used for standard reco (I think).